<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FrmNewLotes</class>
 <widget class="QDialog" name="FrmNewLotes">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>523</width>
    <height>257</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>40</y>
     <width>181</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>REGISTRAR LOTES DE VACUNAS</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>110</y>
     <width>61</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Fecha</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="txtLote">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>110</y>
     <width>161</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>46</x>
     <y>112</y>
     <width>51</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Nombre:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>150</y>
     <width>91</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Cantidad de Lotes</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Grabar">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>210</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Grabar</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Cerrar">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>210</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Cerrar</string>
   </property>
  </widget>
  <widget class="QDateEdit" name="lotFe">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>110</y>
     <width>110</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QSpinBox" name="cantLot">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>160</y>
     <width>42</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
